metpy/plots/declarative.py

```diff
@@ -846,24 +867,41 @@ class ContourPlot(Plot2D):
 
     linecolor = Unicode('black')
     linewidth = Int(2)
-    contours = Union([List(Float()), Int()], default_value=25)
+    linetype = Unicode('solid')
```

Comment:
If we allow `None`, we can get the matplotlib behavior where using a single color for contour will use a different line style for the negative values.
```suggestion
    linetype = Unicode('solid', allow_none=True)
```

Can we can `linetype` to `linestyle` to keep better compatibility with matplotlib? I don't think `linetype` has any inherent better clarity.

Reply:
Done

Reply:
Also fixed test that had this call.